rules/aip0233/resource_reference_type_test.go

```diff
@@ -23,15 +23,17 @@ import (
 func TestResourceReferenceType(t *testing.T) {
 	// Set up testing permutations.
 	tests := []struct {
-		testName string
-		TypeName string
-		RefType  string
-		problems testutils.Problems
+		testName        string
+		TypeName        string
+		RefType         string
+		RepeatedMessage string
```

Comment:
Let's rename this, "RepeatedMessageType", please.

Reply:
Done.